### PR TITLE
RFC: Finalize website

### PR DESCRIPTION
**Pull request by provsalt**
This PR is a **RFC: Requesting for comment**. Please leave your feedback about how the code can be tidied out for production!

TODO:
- [x] Landing page
- [x] Register page 
- [ ] Code cleanup
- [ ] Code refactor
- [ ] Responsive design

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -1,18 +1,34 @@
...
-            <h1 className="text-3xl font-bold">Something&apos;s in the works!</h1>
-            <p className="text-xl font-semibold w-1/3">We&apos;re a group of students planning the biggest hackathon for students in Singapore. Watch this space for more details soon!</p>
-            <p className="w-1/3">Have any questions in the meantime? Shoot us a question at <a href="mailto:team@hacklah.com">team [at] hacklah [dot] com</a>.</p>
-        </main>
-    )
+	return (
+		<>
+			<Head>
+				<title>HackLah! - Singapore teenagers in-person hackathon</title>
+				<meta name="description" content="HackLah! is a hackathon hosted for teenagers in Singapore. Join us in person on 18th March 2023 for fun, workshops, and hacking with like-minded individuals!"/>
```
Comment (cytronicoder) [inline pages/index.tsx:16]: We should change the `meta` description to be in sync with the Instagram account: *HackLah! is a hackathon hosted for teenagers in Singapore. Join us in person on 18th March 2023 for games, workshops, and hacking! 🇸🇬*

**Review by cytronicoder — CHANGES_REQUESTED** [review]
Looks fine so far - thanks a lot!

File: pages/index.tsx (hunk 1 above)
Comment (provsalt) [reply pages/index.tsx]: fixed

**Comment by cytronicoder** [comment]
@provsalt having technical issues right now - imagine not having access

**Comment by cytronicoder** [comment]
I'll take over from here 🚀

**Review by cytronicoder — APPROVED** [review]
LGTM! Next step would be heavy CSS reformatting + mobile optimisation.

File: components/about/about.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+		>
+			<p>
+				HackLah! is a full-day hackathon event hosted by students, for
+				students around the tiny red dot — Singapore. Come together to
+				build projects and share them with the world!
+			</p>
+			<p>
+				Let’s make one thing clear: we don’t mean hack as in gain
+				unauthorized access and take things, we mean create and tinker
+				something new.{" "}
```
Comment (provsalt) [inline components/about/about.tsx:19]:
```suggestion
				something new.
```

**Review by provsalt — COMMENTED** [review]
was this meant to be here?

File: components/about/about.tsx (hunk 2 above)
Comment (cytronicoder) [reply components/about/about.tsx:19]: Hmm someone put a space character there 🤨

Comment (cytronicoder) [reply components/about/about.tsx]: Fixed!

**Review by cytronicoder — APPROVED** [review]
LGTM! Ship it 🚀

**Pull request merged**